client/src/modules/stock/adjustment/adjustment.html

```diff
@@ -35,123 +35,121 @@
+            <p translate>STOCK.INCREASE_HELP</p>
+          </div>
 
-      <div class="col-xs-6">
-        <div ng-if="StockCtrl.adjustmentOption === 'increase'" style="margin-top: 0px;">
-          <h4><i class="fa fa-arrow-circle-up"></i> <span translate>STOCK.INCREASE</span> </h4>
-          <p translate>STOCK.INCREASE_HELP</p>
-        </div>
-
-        <div ng-if="StockCtrl.adjustmentOption === 'decrease'" style="margin-top: 0px;">
-          <h4><i class="fa fa-arrow-circle-down"></i> <span translate>STOCK.DECREASE</span> </h4>
-          <p translate>STOCK.DECREASE_HELP</p>
+          <div ng-if="StockCtrl.adjustmentOption === 'decrease'" style="margin-top: 0px;">
+            <h4><i class="fa fa-arrow-circle-down"></i> <span translate>STOCK.DECREASE</span> </h4>
+            <p translate>STOCK.DECREASE_HELP</p>
+          </div>
         </div>
       </div>
-    </div>
-
-    <!-- add and recovery -->
-    <div class="row" style="padding-bottom : 5px;">
-      <div class="col-xs-12">
-        <!-- info -->
-        <p
-          style="padding-top : 10px;"
-          class="text-info"
-          ng-show="!StockCtrl.adjustmentOption">
-          <span class="fa fa-info-circle"></span>
-          <span translate>FORM.INFO.NO_DESTINATION</span>
-        </p>
-
-        <!-- "Recover Items" button -->
-        <div class="btn-group pull-right" role="group">
-          <button
-            type="button"
-            class="btn btn-default"
-            ng-class="{'btn-primary' : StockCtrl.hasCacheAvailable }"
-            ng-click="StockCtrl.readCache()"
-            ng-disabled="!StockCtrl.hasCacheAvailable">
-            <span class="fa fa-recycle"></span>
-            <span translate>FORM.BUTTONS.RECOVER_ITEMS</span>
-          </button>
-        </div>
 
-        <!-- "Add number of grid rows" input-group -->
-        <bh-add-item
-          disable = "StockCtrl.adjustmentOption"
-          callback = "StockCtrl.addItems(numItem)">
-        </bh-add-item> 
+      <!-- add and recovery -->
+      <div class="row" style="padding-bottom : 5px;">
+        <div class="col-xs-12">
+          <!-- info -->
+          <p
+            style="padding-top : 10px;"
+            class="text-info"
+            ng-show="!StockCtrl.adjustmentOption">
+            <span class="fa fa-info-circle"></span>
+            <span translate>FORM.INFO.NO_DESTINATION</span>
+          </p>
+
+          <!-- "Recover Items" button -->
+          <div class="btn-group pull-right" role="group">
+            <button
+              type="button"
+              class="btn btn-default"
+              ng-class="{'btn-primary' : StockCtrl.hasCacheAvailable }"
+              ng-click="StockCtrl.readCache()"
+              ng-disabled="!StockCtrl.hasCacheAvailable">
+              <span class="fa fa-recycle"></span>
+              <span translate>FORM.BUTTONS.RECOVER_ITEMS</span>
+            </button>
```

Comment:
[not blocking] : this button doesn't do anything as long as there is not any implementation of read and write into the cache for data defined in this page.